### PR TITLE
fix(ui): correct wrong profile linking from posts on bookmark page

### PR DESCRIPTION
**Pull request by ghost**
## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/plugins/bookmarks/src/components/bookmarks-page.tsx [hunk 1]
```diff
@@ -52,26 +54,23 @@ const BookmarksPage: React.FC<BookmarksPageProps> = props => {
...
-      }
-      navigateToUrl(url);
-    };
-
-  const handleNavigateToPost = redirectToPost(props.singleSpa.navigateToUrl);
+  const handleNavigation = (itemType: ItemTypes, details: IContentClickDetails) => {
+    let url;
+    switch (itemType) {
+      case ItemTypes.PROFILE:
+        url = `/profile/${details.entryId}`;
```
Comment (kenshyx) [inline ui/plugins/bookmarks/src/components/bookmarks-page.tsx:61]: all the links are built with `details.entryId` ? O_O

Comment (ghost) [reply ui/plugins/bookmarks/src/components/bookmarks-page.tsx:61]: Uhhhhm 😅 It is misdirecting, but it's the same pattern used elsewhere in the repo. I could change it to just ie. id everywhere?

Comment (quininez) [reply ui/plugins/bookmarks/src/components/bookmarks-page.tsx:61]: It's a generic handler for multiple types of redirects, and unless we refactor all of it I wouldn't change it now

Comment (kenshyx) [reply ui/plugins/bookmarks/src/components/bookmarks-page.tsx:61]: we should create a todo to refactor this ;/

**Review by kenshyx — APPROVED** [review]

**Pull request merged**